### ISSUE TITLE
Booklet folded in 4 (not to)

### ISSUE THREAD

**Issue by csillag** [Boomaga/boomaga#54]
Currently, boomaga supports printing booklets printing 2 pages on one (each side of) sheet, and then folding in into two.

That's great - however, there are situations when one wants to create a booklet by folding the printed sheet not into 2, but into 4 or even 8. (And then cutting the edges, as it is done in traditional book binding.) The page imposition required for this is similar to 4-up (or 8-up), but not exactly the same.

It would be great if Boomaga could support that, too.

(PDFjam used to be able to handle these use cases, but it has been broken for a few years now, and it seems to have been abandoned...)

Thanks.

**Comment by SokoloffA** [comment]
I agree, it is sometimes necessary. But only a bit later. First I have to fix a [serious bug](https://github.com/flacon/flacon/issues/48) in another my project.

**Comment by csillag** [comment]
Yes. Meanwhile, I have found another solution, here: https://github.com/dgoodmaniii/makebook
It also missed some of the features I needed, but I could add them easily...

So, no pressure from me, I'm good for now :)
